app/wsdl2rest/pom.xml

```diff
@@ -0,0 +1,149 @@
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" 
+    xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>wsdl2rest-impl-fatjar</artifactId>
+    <name>Wsdl2Rest Fat Jar</name>
+    <version>0.1.3-SNAPSHOT</version>
+    <groupId>org.github.camel.tooling.wsdl2rest</groupId>
+
+    <properties>
+        <version.wsdl2rest>0.8.0.fuse-720034</version.wsdl2rest>
```

Comment:
double-checking, it is the version that has been pinned for non-deletion even if initially in staging repo?

Reply:
Yes, this is the version that is pinned